pages/news/2020-05-15-you-dont-need-to-pay-for-mixxx.html

```diff
@@ -0,0 +1,76 @@
+Instead, you can download the latest version of Mixxx from the [download page on our website]({% url "/download.html" %}) for free.
+
+If you want to support Mixxx financially, consider [donating](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=QSFMYWN2B3JD2&source=url) - we'll appreciate it!
+
+
+### Mixxx was, is and always will be free
+
+In contrast our commercial competitors that offer "free" featured-reduced versions of their Software, the Mixxx developers do not impose any artificial restrictions on their software.
+There is no "Pro Edition" or Premium version of Mixxx, even if it says so in the description of the online shop.
+
+The Mixxx developers don't sell Mixxx, and if you "buy" Mixxx somewhere online you'll likely just get the free Mixxx version that the seller downloaded from our website and burned onto a CD.
+Even worse, some of these paid Mixxx versions might even contain ads or malware according to Amazon reviews.
+Just use the free [download]({% url "/download.html" %}) from our website to get the ad-free, malware-free, most recent version of Mixxx.
+
+
+### Mixxx is free as in freedom, not just free as in "free beer"
+
+You don't need to pay money to use Mixxx, but there's more:
+Its source code is accessible to everyone and if you know how to code, you can modify it and customize Mixxx to a degree that is simply not possible with any closed-source software.
+Even if you did get your hands on the source code of Serato DJ Pro/Lite, Rekordbox, Traktor or VirtualDJ (which is not available their users), it would be illegal for you to modify the program or redistribute it.
+
+Mixxx, on the other hand, is [free software](https://www.gnu.org/philosophy/free-sw.html) and grants you the four essential freedoms:
+
+- The freedom to run the program as you wish, for any purpose.
+- The freedom to study how the program works, and change it so it does your computing as you wish. Access to the source code is a precondition for this.
+- The freedom to redistribute copies so you can help others.
+- The freedom to distribute copies of your modified versions to others. By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
+
+These freedoms are part of the license that Mixxx is subject to - the [GNU General Public License (GPL)](https://github.com/mixxxdj/mixxx/blob/master/LICENSE).
+If you want to know more about Mixxx' license, [check out the FAQ](https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq).
+
+
+### Getting involved
+
+The Mixxx project is entirely community-driven and supported by volunteers from all over the globe.
+None of our developers are paid - all of us are passionate about the project and and work on Mixxx in our spare-time.
+
+We dedicate our time to working Mixxx, because we believe in open-source and care about delivering a decent product to our users - including ourselves!
```

Comment:
```suggestion
We dedicate our time to working Mixxx, because we believe in open-source and care about delivering a decent product to our users — including ourselves!
```